VultisigApp/VultisigApp/Model/Coin.swift

```diff
@@ -173,6 +175,8 @@ class Coin: ObservableObject, Codable, Hashable {
             return "180000"
         case .akash:
             return "200000"
+        case .tron:
+            return "100000" // TODO: double check this
```

Comment:
do you still need to double check? It seems to me we can't hardcode the fee, as fee depends on multiple factors on TRON chain